cognite/neat/_rules/importers/_rdf/_inference2rules.py

```diff
@@ -26,7 +26,11 @@
                            WHERE { ?s a ?class . }
                            group by ?class order by DESC(?instances)"""
 
-INSTANCES_OF_CLASS_QUERY = """SELECT ?s WHERE { ?s a <class> . }"""
+
+INSTANCES_OF_CLASS_QUERY = """SELECT ?s (COUNT(?p) as ?propertyCount)
```

Comment:
main 🧠 , a bit slower query than the previous, but here you can really count on getting the most richest instances first.

Reply:
SMART !!!  🧠 🧠 🧠 🧠 🧠